website/docs/docs/on-premises/releases/2.1.0.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 2.1.0
+---
+
+## Release date
+
+10 July 2025
+
+## Features
+
+- Added supports and verify sensitive information to be obfuscated when sending to 
+OpenTelemetry
+- Enhanced tracing capabilities across distributed systems
+- Major refactor in the UI to improve page loading time
+- Updated HAL Explorer for enhanced functionality
```

Comment:
what's the functionality?

Reply:
That should not be enhanced functionality. It should be dependencies upgrade to fix security vulnerabilities